example/django/bugsnag_demo/settings.py

```diff
@@ -38,7 +38,8 @@
     'django.contrib.staticfiles',
 )
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = (
+    # make sure to add Bugsnag to your middleware.
```

Comment:
While this step is mentioned in the getting started docs, I think it can be overlooked and it may deserve a step in the readme.